### ISSUE TITLE
RELEASE: Banner component

### ISSUE THREAD

**Issue by Whersi** [ministryofjustice/moj-frontend#1018]
## Pre-release checklist

- [x] Content: prepared content for What’s new section of documentation website
- [x] Content: prepared Slack post
- [x] Content: prepared MoJ FE release note - M
- [x] Content: prepared Figma release note
- [x] Content: prepared Notify content
- [x] Team: agreed Slack post, Notify email and release notes

## Decision point
- [x] [Check the building block against the MoJ Design System standards](https://github.com/ministryofjustice/moj-frontend/issues/853)
- [x] Team decision point: is the release ready?

## Release checklist
- [x] Interaction: Merge component branch to main MoJ Figma Kit
- [x] Developer: Commit the changes to MoJ FE
- [x] Developer: Publish changes from staging to documentation site (inc. release notes)
- [x] Team: Check published version on a call (to start no later than 3pm on day of release)
- [x] Interaction: Publish update on Figma (inc. release notes)
- [x] Developer: Commit homepage content
- [x] Content: Send Slack post
- [x] Content: Send Notify email

## Post-release checklist

- [ ] DM: Post the release notes in the Github discussion 
- [ ] Lead: Add building block to analytics spreadsheet

**Comment by robertjmccarthy** [comment]
The team are working on a release strategy so the plan is to do all the work for the banner component to have it ready for release, but not actually release it. Once the release strategy is in place then we can release the improved banner component.

**Comment by murrlipp** [comment]
@robertjmccarthy as we're now heading for a release on 24th Feb when I'm not here, it would be helpful if you could do the following activities as part of the release. They need to be done before the [banner-updates git branch](https://github.com/ministryofjustice/moj-frontend/tree/banner-updates) is merged into main.

1. Double check the ['Alert' Figma branch](https://www.figma.com/design/N2xqOFkyehXwcD9DxU1gEq/branch/MhpA8RbhYwYpr40kUvi03S/MoJ-Figma-Kit?m=auto) before merging:
    - [ ] Check that the dates on the 'Alert' and 'Banner' pages match the release date.
    - [ ] Check that the relevant [icons](https://www.figma.com/design/MhpA8RbhYwYpr40kUvi03S/alert?node-id=877-224&t=fiw9JEVJXH40Mmne-4) for the Alert component are set to 'hide when publishing. This can be done by going to the assets panel, finding the components under `Created in this file > _Building blocks`, right-clicking each one and selecting 'Hide when publishing'.

    The above should all be fine but it's worth double checking before merging anything!

2. Merge the branch.
3. Copy the URL of the Alert page which should now be in the [MoJ Figma Kit](https://www.figma.com/design/N2xqOFkyehXwcD9DxU1gEq/MoJ-Figma-Kit?m=auto&node-id=3316-981&t=8CpaEzSD9FFjy2eA-1) file.
4. Add the URL to all of the examples in the [banner-updates git branch](https://github.com/ministryofjustice/moj-frontend/tree/banner-updates).
    - [ ] docs/examples/alert/index.njk
    - [ ] docs/examples/alert-dismissible/index.njk
    - [ ] docs/examples/alert-error/index.njk
    - [ ] docs/examples/alert-information/index.njk
    - [ ] docs/examples/alert-success/index.njk
    - [ ] docs/examples/alert-warning/index.njk

5. Publish the [MoJ Figma Kit](https://www.figma.com/design/N2xqOFkyehXwcD9DxU1gEq/MoJ-Figma-Kit?m=auto&node-id=3316-981&t=8CpaEzSD9FFjy2eA-1).
